samples/highcharts/accessibility/accessible-switch-theme/demo.js

```diff
@@ -0,0 +1,355 @@
+// Light and dark themes
```

Comment:
When first loading the page, the default Highcharts theme seems to be loaded - instead of the light theme